samples/python/console/speech_sample.py

```diff
@@ -288,6 +288,7 @@ def stop_cb(evt):
     speech_recognizer.stop_continuous_recognition()
     # </SpeechContinuousRecognitionWithFile>
 
+# <SpeechRecognitionFromMicrophone>
```

Comment:
Seems this tag should have keyword in it since it's the keyword sample.

Reply:
@rhurey - Updated. Thanks